runner.py

```diff
@@ -104,8 +104,12 @@ def take_result_median(result_json, expected_runs):
             if k == "testcase":
                 median_result[k] = group[0][k]
             else:
-                median_result[k] = median(filter(lambda x: x is not None,
-                                                 map(lambda x: x[k], group)))
+                try:
+                    median_result[k] = median(
+                            filter(lambda x: x is not None,
+                                   map(lambda x: x[k], group)))
+                except StatisticsError:
+                    median_result[k] = 0
```

Comment:
Let's use `-1` to indicate an error. I know it's dirty, but that is easier to spot on Perfherder.